src/main/java/frc/robot/subsystems/coral/CoralEffectorConstants.java

```diff
@@ -17,7 +17,7 @@ public class CoralEffectorConstants {
   public static final TunableNumber CORAL_HOLD_PERCENT =
       new TunableNumber("CoralEffector/HoldPercent", 0.1);
   public static final TunableNumber EFFECTOR_PULLBACK_SECONDS =
-      new TunableNumber("CoralEffector/PullbackTime", 0.1);
+      new TunableNumber("CoralEffector/PullbackTime", 2);
```

Comment:
Let’s keep doubles consistent with having a decimal place
```suggestion
      new TunableNumber("CoralEffector/PullbackTime", 2.0);
```